### PR TITLE
ft[update user profile]

### PR DESCRIPTION
**Pull request by NIRoberto**
What does this PR do?

- For authentication should be able to update his/her profile information

Description of Task completed

- User should be able to provide his or her profile info in in the provided field 

How should this be manually tested?

- git clone to https://github.com/atlp-rwanda/elite-bn-frontend.git
- checkout to ft-update-user-profile
- `npm install` 
- `npm start` to start the server or `npm run dev`
- Login by providing the email and password 
- And then after login user should be able t view his/her profile info
- On below profile info click on button for edit profile
- fill in the form the new field and click update to update user info
What is the relevant Trello story?

-https://trello.com/c/O4eKW9OE

### CODE REVIEW

File: src/Routes/index.js [hunk 1]
```diff
@@ -10,10 +12,11 @@ function index() {
     <Switch>
       <Route path="/register" component={Auth} />
       <Route exact path="/" component={Home} />
-
+      <Route exact path="/dashboard/profile/" component={SingleProfile} />
       <Route exact path="/login" component={login} />
 
-      <Route exact path="/dashboard" component={DashBoard} />
+      <Route exact path="/dashboard" component={DashBoards} />
```
Comment (nicele08) [inline src/Routes/index.js]: no need to to make it exact
we will use it to render other components on it.

**Review by nicele08 — COMMENTED** [review]
Well done

**Review by iradukundairenee — APPROVED** [review]

**Review by nicele08 — APPROVED** [review]

**Comment by pextech** [comment]
can you try and fix your conflicts @NIRoberto

**Review by 444root — APPROVED** [review]

File: src/test/components/DashBoard.test.js [hunk 2]
```diff
@@ -1,18 +1,49 @@
 import React from 'react';
 import { shallow } from 'enzyme';
+<<<<<<< HEAD
```
Comment (444root) [inline src/test/components/DashBoard.test.js]: remove these rebase lines ,

**Pull request merged**